<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticleAddForm</class>
 <widget class="QWidget" name="ParticleAddForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a particle</string>
  </property>
  <widget class="QLabel" name="m_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Mass:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pos_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Position:</string>
   </property>
  </widget>
  <widget class="QLabel" name="vel_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Velocity:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mass_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="posx_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="posy_edit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="posz_edit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="vely_edit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="velx_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="velz_edit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="x_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>X axis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="y_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Y axis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="z_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Z axis</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="preview_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Preview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="note_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: Preview uses euler method</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="tag_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tag (Optional):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tag_edit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="error_hint">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>111</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
